tools/cbmc/proofs/Queue/QueueGenericCreateStatic/QueueGenericCreateStatic_harness.c

```diff
@@ -0,0 +1,30 @@
+#include "FreeRTOS.h"
+#include "queue.h"
+#include "queue_datastructure.h"
+#include "cbmc.h"
+
+
+void harness(){
+    UBaseType_t uxQueueLength;
+    UBaseType_t uxItemSize;
+    uint8_t *ucQueueStorage = (uint8_t *) pvPortMalloc(sizeof(uint8_t));
+
+    //The queue storage space is explicitly assumed to be not null.
+    StaticQueue_t xStaticQueue;
+    uint8_t ucQueueType;
+
+    /* The QueueGenericReset method called during static queue creation
+       implicitly that there are no overflows in the computation
+       and the inputs are safe. There is no check for this in the code base. */
+    UBaseType_t upper_bound = portMAX_DELAY - sizeof(Queue_t);
```

Comment:
I asked this in a previous PR.  portMAX_DELAY is an unclear name for the macro. Can you create a descriptive alias  for it